### PR TITLE
Milestone: Create a database based on a diagram

### PR DESCRIPTION
**Pull request by sanieni6**
# Milestone: Create a database based on a diagram
This pull request includes the following changes:
- Add tables according the provided diagram
- Create relationships according the provided diagram
## Team Members:
@NicolasAndrehh 
@sanieni6

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+REFERENCES patients (id);
+
+ALTER TABLE invoice_items
+ADD CONSTRAINT FK_invoices_items_invoices
+FOREIGN KEY (invoice_id)
+REFERENCES invoices (id);
+
+ALTER TABLE invoice_items
+ADD CONSTRAINT FK_invoice_items_treatments
+FOREIGN KEY (treatment_id)
```
Comment (nedjwab) [inline schema_based_on_diagram.sql:63]: [ ] kindly  Create the indexes for all your foreign keys as it's part of the project requirements.😊
👇👇👇
req

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @NicolasAndrehh    

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the project requirements, but you still have some issues: 
check the comments below 
 👇👇👇

 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by NicolasAndrehh** [comment]
Thank you so much for your feedback @nedjwab!
We're going to implement it right now! 😄👍

**Review by danmainah — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ Added all foreign keys correctly
✔️ Github flow was correctly used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**